### ISSUE TITLE
Tests are not included in eslint `parserOptions.project`

### ISSUE THREAD

**Issue by ChocolateLoverRaj** [imbhargav5/rooks#533]
**Describe the bug**
I am using [the eslint vscode plugin](https://github.com/Microsoft/vscode-eslint).

```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: src\__tests__\useMapState.spec.tsx.
The file must be included in at least one of the projects provided.
```

**To Reproduce**
Steps to reproduce the behavior: Install the plugin
1. Go to any `src/__tests__/*.spec.tsx` file.
2. Go to first character of file.
3. See error from the extension

**Expected behavior**
The eslint extension should be able to show / fix linting errors on `.spec.tsx` files.

**Screenshots**
![error](https://user-images.githubusercontent.com/52586855/127694337-5f3ab704-8d4c-44b4-aa0e-fd678d1c00f4.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: `main` branch

- [x] I can make a PR fixing this

**Comment by imbhargav5** [comment]
Strange. I thought this was fixed 🤔. 

Please submit a PR . That would be 🔥🔥

**Comment by imbhargav5** [comment]
Fixed in #536
